Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -395,11 +395,11 @@
     impactEffect: BulletImpactEffectKinetic
     damage:
       types:
-        Blunt: 25
-        Structural: 30
+        Blunt: 15
+        Structural: 10
   # Short lifespan
   - type: TimedDespawn
-    lifetime: 0.4
+    lifetime: 0.170 #the cost of infinite ammo. goes about 4~ tiles
   - type: GatheringProjectile
```

Comment:
TimedDespawn is not a good indicator of range when we change bullet speeds.

Reply:
PKA changes have now been removed from the PR